app/config/parameters.yml.dist

```diff
@@ -16,6 +16,7 @@ parameters:
     twitter_consumer_secret: "ChangeMe"
     twitter_access_token: "ChangeMe"
     twitter_access_token_secret: "ChangeMe"
+    twitter_screen_name: "010php"
```

Comment:
I'd prefer not to configure it unless there is a use case where this is changed based on the environment.